### ISSUE TITLE
Homework4 Q1 实际更改了代码, 不需要提交 diff 吗?

### ISSUE THREAD

**Issue by yeoyi519** [PeiqinSun/tf-tutorials#72]

**Comment by ChinaYi** [comment]
同问，实际上，代码里面使用的已经是Q2中teacher 网络输出的logits了，如下：
`target_labels = teacher_network.get_prediction(images) ## assign teacher label here
 feed_dict = {
                    placeholders['data']: images,
                    placeholders['label']: labels,
                    target_label_onehot: target_labels,
                    global_steps: global_cnt,
                    placeholders['is_training']: True,
}`
而我们要实现Q1，用onehot label代替teacher label反而需要改动代码。

**Comment by RayXu14** [comment]
我在网站上提交的时候不需要q1的diff就过了

**Comment by kifish** [comment]
用01-svhn里的代码。
